### DISCUSSION TITLE
Load audio in ASR

### DISCUSSION THREAD

**Discussion by talhaanwarch** [huggingface/datasets#4265] [General]
Hi, I am following the documentation of ASR wave2vec tuning [here](https://huggingface.co/blog/fine-tune-xlsr-wav2vec2) and [here](https://huggingface.co/blog/fine-tune-wav2vec2-english)
In the documentation is written 

> We can see that the audio file has automatically been loaded. This is thanks to the new ["Audio" feature](https://huggingface.co/docs/datasets/package_reference/main_classes.html?highlight=audio#datasets.Audio) introduced in datasets == 4.13.3, which loads and resamples audio files on-the-fly upon calling.

transformers.__version__: '4.12.5'
datasets.__version__:'2.1.0'

This is the code
```
from datasets import load_dataset
data_files = {"train": "../train.tsv", "val": "../dev.tsv"}
urdata = load_dataset("csv", delimiter='\t',data_files=data_files)

def relapth(batch):
    return {"path":'/'.join(['../clips',batch['path']]) }
urdata=urdata.map(relapth)
```

Now how can i load audio? In tutorial/docs, the audio is automatically uploaded. But in my case its not. How can i do that .
Currently i am doing 
```
import librosa
def loadaudio(batch):
    return {"audio":librosa.load(batch['path'],sr=16000) }
urdata=urdata.map(loadaudio)
```

**Comment by mariosasko** [comment]
Hi! You can get an audio column from your column of audio paths as follows:
```python
urdata = urdata.cast_column("path", datasets.Audio(sampling_rate=16000))
```
